wordpress-mobile/WordPress-Android
# Replace some vector assets with PNG's

Fixes #5560 

Discussion in #5584 led to a ~different~ _much more efficient_ approach in this PR. We're replacing some of the vector icons with PNG's to prevent crashes with API < 21 devices.

I went through the layout files changed in the previous PR and is seems like they fall into three buckets:

1. Keep layout as-is because `drawable*` values are not vector assets (yay, less diff!)
	* photo_chooser_fragment
	* post_location_settings_add
	* post_location_settings_view
	* reader_fragment_post_detail
	* reader_tag_header_view
	* stats_visitors_and_views_fragment
	* category_button
	* category_select_button
	* my_site_fragment
	* page_item

2. Use `app:drawable*Compat` due to a potential for for a vector asset (only requires `WPTextView` changes, no new classes!)
	* filter_spinner_item : use `drawableRightCompat` because `ic_dropdown_blue_light_24dp` potentially uses a vector asset
	* people_invite_fragment : use `drawableRightCompat` because `ic_dropdown_blue_light_24dp` potentially uses a vector asset

3. Add PNG assets to use as drawable values to avoid needing custom View classes (some new PNG assets, nbd?)
	* media_gallery_settings_fragment : `ic_gallery_*_selector` are vectors, add PNG assets for CheckBox's
		* ic_gallery_grid_selector
		* ic_gallery_tiled_selector
		* ic_gallery_squares_selector
		* ic_gallery_circles_selector
		* ic_gallery_slideshow_selector

To test:
* Run the app with API < 5.0 and >= 5.0
* Check media gallery settings, Stats, and People invite screens
* Verify that the icons are showing correctly
